### ISSUE TITLE
Create account management tool

### ISSUE THREAD

**Issue by nemerle** [Segs/Segs#34]
Minimal functionality:
- Account Creation
- Account Removal
- Account Search/Edit

A clean database schema + access description should allow creation of simple website with this functionality

**Comment by broxen** [comment]
We're almost there with @dracc and the team's work on SEGSAdmin + the web-ui over at segs-data

**Comment by dracc** [comment]
I guess accounts with access level 9 ought to have the ability to remove accounts, as well as users ought to be able to remove their own accounts.
But what would search/edit do? I can see `password change` being a valuable function. Are there more functionality expected that I just fail to see?

**Comment by broxen** [comment]
Should users be able to remove their own account though? What if we allow them to disable it, and provide the admin with a way to purge disabled accounts (perhaps periodically)

My thought is that users are some times impetuous and having the ability to "undo" for a brief period of time seems good.

**Comment by broxen** [comment]
For Search and Edit:

**Search:** Find a user by User ID, by db_id, by IP address (which we should be logging) or by player name. Also maybe filter by last login date?

**Edit:** Edit should allow us to reset user passwords, update contact email, edit character information (char name, in particular, but probably other aspects as well)

The greatest use I see for these abilities would be to revise characters with offensive names, or who have managed to cheat in some way (level 69? I don't think so!)

I would also add to this list the ability to **Disable** or **Ban** accounts.

**Comment by mobbyg** [comment]
I agree. They may decide to come back later. And while many will be ok with all their stuff gone, because they KNEW what they were doing, others may just not be as... observant. 

Perhaps a drop down along with a search field to find the users, and a set of tick boxes to do some basic admin like delete or suspend etc. Even the ban feature?

**Comment by broxen** [comment]
This relates to #436
